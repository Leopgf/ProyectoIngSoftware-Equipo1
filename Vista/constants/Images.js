// local imgs
const Onboarding = require('../assets/imgs/FFondo.jpg');
const Logo = require('../assets/imgs/logop.png');
const ProfileBackground = require('../assets/imgs/basePerfil.jpg');
const RegisterBackground = require('../assets/imgs/fondoInicio.jpg');
const Pro = require('../assets/imgs/bg.png');
const NowLogo = require('../assets/imgs/logo.png');
const iOSLogo = require('../assets/imgs/ios.png');
const androidLogo = require('../assets/imgs/android.png');
const ProfilePicture = require('../assets/imgs/profile-img.jpg');
const Conversor = require('../assets/imgs/conversor.jpg');
const AddReceta = require('../assets/imgs/addReceta.jpg');
const CreativeTimLogo = require('../assets/imgs/creative-tim-white-slim2.png');
const InvisionLogo = require('../assets/imgs/invision-white-slim.png');

const Viewed = [
  require('../assets/imgs/project5.jpg'),
  require('../assets/imgs/project7.jpg'),
  require('../assets/imgs/project6.jpg'),
  require('../assets/imgs/project4.jpg'),
  require('../assets/imgs/project21.jpg'),
  require('../assets/imgs/project24.jpg')
];

const Products = {
  text: 'View article',
  path: require('../assets/imgs/bg15.jpg')
};

export default {
  Onboarding,
  Logo,
  ProfileBackground,
  Conversor,
  ProfilePicture,
  RegisterBackground,
  AddReceta,
  Viewed,
  Pro,
  Products,
  NowLogo: NowLogo
};
