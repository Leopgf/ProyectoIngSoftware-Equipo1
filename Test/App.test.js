import React from 'react';
import renderer from 'react-test-renderer';

import { Block, GalioProvider } from 'galio-framework';
import { NavigationContainer } from '@react-navigation/native';
import { nowTheme } from '../Vista/constants';

import { AppLoading } from 'expo';

describe('<AppLoading />', () => {
  //TEST PANTALLA CARGA
    it('renders correctly', () => {
        const tree = renderer.create(< AppLoading />).toJSON();
        expect(tree).toMatchSnapshot();
      });
  // TEST NAVEGACION
    it('renders correctly', () => {
        const tree = renderer.create(
        <NavigationContainer>
          <GalioProvider theme={nowTheme}>
            <Block flex>
            </Block>
          </GalioProvider>
        </NavigationContainer>
        ).toJSON();
        expect(tree).toMatchSnapshot();
      });
});