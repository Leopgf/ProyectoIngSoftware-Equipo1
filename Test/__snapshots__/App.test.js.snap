// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppLoading /> renders correctly 1`] = `null`;

exports[`<AppLoading /> renders correctly 2`] = `
<View
  style={
    Array [
      Object {
        "flexDirection": "column",
      },
      false,
      Object {
        "flex": 1,
      },
      false,
      false,
      false,
      false,
      false,
      false,
      null,
      false,
      false,
      false,
      null,
      null,
      null,
      undefined,
    ]
  }
  theme={
    Object {
      "COLORS": Object {
        "ACTIVE": "#f96332",
        "AZULFUERTE": "#0F1E2E",
        "BEHANCE": "#1769ff",
        "BLACK": "#0f1e2e",
        "BLOCK": "#E7E7E7",
        "BORDER": "#0f1e2e",
        "BORDER_COLOR": "#0f1e2e",
        "BUTTON_COLOR": "#9C26B0",
        "DEFAULT": "#7faac7",
        "DRIBBBLE": "#ea4c89",
        "ERROR": "#FF3636",
        "FACEBOOK": "#3b5998",
        "GITHUB": "#333333",
        "GOOGLE": "#dd4b39",
        "GRADIENT_END": "#AC2688",
        "GRADIENT_START": "#6B24AA",
        "GREY": "#898989",
        "HEADER": "#0f1e2e",
        "ICON": "#172B4D",
        "ICON_INPUT": "#555555",
        "INFO": "#2CA8FF",
        "INPUT": "#DCDCDC",
        "INPUT_ERROR": "#ff3636",
        "INPUT_SUCCESS": "#1be611",
        "LABEL": "#FE2472",
        "LINKEDIN": "#0077B5",
        "MUTED": "#8898AA",
        "NAVBAR": "#F9F9F9",
        "NEUTRAL": "rgba(255, 255, 255, 0.2)",
        "PINTEREST": "#cc2127",
        "PLACEHOLDER": "#9FA5AA",
        "PRICE_COLOR": "#EAD5FB",
        "PRIMARY": "#b8d9f4",
        "PURE_WHITE": "#f5f5f5",
        "REDDIT": "#ff4500",
        "SECONDARY": "#0f1e2e",
        "SUCCESS": "#18ce0f",
        "SWITCH_OFF": "#898989",
        "SWITCH_ON": "#f96332",
        "TABS": "rgba(222, 222, 222, 0.3)",
        "TEXT": "#0f1e2e",
        "THEME": "#B23AFC",
        "THUMB_SWITCH_ON": "#fff",
        "TRANSPARENT": "transparent",
        "TUMBLR": "#35465c",
        "TWITTER": "#55acee",
        "WARNING": "#FFB236",
        "WHITE": "#8b8c89",
        "YOUTUBE": "#e52d27",
      },
      "SIZES": Object {
        "ANDROID_ELEVATION": 1,
        "BASE": 16,
        "BLOCK_SHADOW_OPACITY": 0.15,
        "BLOCK_SHADOW_RADIUS": 8,
        "BORDER_RADIUS": 6,
        "BORDER_WIDTH": 0.8,
        "BUTTON_HEIGHT": 44,
        "BUTTON_SHADOW_RADIUS": 10,
        "BUTTON_WIDTH": 144,
        "CARD_AVATAR_HEIGHT": 40,
        "CARD_AVATAR_RADIUS": 20,
        "CARD_AVATAR_WIDTH": 40,
        "CARD_BORDER_RADIUS": 6.4,
        "CARD_BORDER_WIDTH": 0.8,
        "CARD_FOOTER_HORIZONTAL": 12,
        "CARD_FOOTER_VERTICAL": 12,
        "CARD_IMAGE_HEIGHT": 200,
        "CARD_MARGIN_VERTICAL": 14,
        "CARD_ROUND": 3,
        "CARD_ROUNDED": 8,
        "CARD_WIDTH": 718,
        "CHECKBOX_HEIGHT": 20,
        "CHECKBOX_WIDTH": 20,
        "FONT": 16,
        "ICON": 16,
        "INPUT_BORDER_RADIUS": 8,
        "INPUT_BORDER_WIDTH": 0.8,
        "INPUT_HEIGHT": 44,
        "INPUT_HELP_TEXT": 12.8,
        "INPUT_HORIZONTAL": 16,
        "INPUT_LABEL_BOTTOM": 4,
        "INPUT_LABEL_TEXT": 14.4,
        "INPUT_ROUNDED": 27.2,
        "INPUT_TEXT": 14,
        "NAVBAR_HEIGHT": 66,
        "NAVBAR_LEFT_FLEX": 0.5,
        "NAVBAR_LEFT_HEIGHT": 93.38000000000001,
        "NAVBAR_LEFT_MARGIN": 16,
        "NAVBAR_RIGHT_FLEX": 0.5,
        "NAVBAR_RIGHT_HEIGHT": 93.38000000000001,
        "NAVBAR_RIGHT_MARGIN": 16,
        "NAVBAR_TITLE_FLEX": 2,
        "NAVBAR_TITLE_HEIGHT": 93.38000000000001,
        "NAVBAR_TITLE_TEXT": 14,
        "NAVBAR_VERTICAL": 16,
        "OPACITY": 0.8,
        "RADIO_HEIGHT": 24,
        "RADIO_THICKNESS": 2,
        "RADIO_WIDTH": 24,
        "THUMB_SIZE": 25,
        "TRACK_SIZE": 4,
      },
    }
  }
/>
`;
